libpius/util.py

```diff
@@ -84,6 +91,8 @@ def parse_dotfile(parser):
       parts = sep.split(line.strip())
       if not parts[0].startswith('--'):
         parts[0] = '--%s' % parts[0]
+      if parts[0] in EXPAND_USER_OPTIONS and len(parts) > 1:
+        parts[1] = os.path.expanduser(parts[1])
       if parser.has_option(parts[0]):
         opts.extend(parts)
       elif not parts[0] in VALID_OPTIONS:
```

Comment:
I know I wrote this code, but... I also, dunno what the point is.

If it's a valid option, then `parser.has_option` will be true, and if not, it's not a valid option... so we should get ride of `VALID_OPTIONS` altogether, and just make this an `else`, I think... you don't have to do that in this PR, just... happened to notice this.